Developer-Guide.md

```diff
@@ -35,13 +36,30 @@ This document is written **specifically for developers**.
 - You already have the following installed:
   - [Docker](https://www.docker.com/).
   - [golang](https://golang.org/dl) version 1.8.3 or newer.
+
+     To view the versions of go known to work, see the `golang` entry in the
+     [versions database](https://github.com/kata-containers/runtime/blob/master/versions.yaml).
+
   - `make`.
   - `gcc` (required for building the shim and runtime).
+  - `qemu` version 2.11 or newer.
+
+     Note that `qemu` version 2.10 may also work but will not perform as well
+     as `qemu` version 2.11 or newer.
+
+     If your distribution does not yet provide a new enough version of `qemu`,
+     you can install an optimised version of `qemu` called `qemu-lite`. This
+     package is installed automatically when you install Clear Containers, but can
+     be installed separately as well:
+
+       https://github.com/clearcontainers/runtime/wiki/Installation
+
+     However, note that the use of `qemu-lite` is deprecated in favour of
```

Comment:
Suggest replacing "favour" with "favor": @jodh-intel maybe we should have @rcaballeromx make a call on whether we go with American or British English as a rule for instances moving ahead?

Reply:
American English!  :)